Formula/zoxide.rb

```diff
@@ -0,0 +1,17 @@
+class Zoxide < Formula
+  desc "Shell extension to navigate your filesystem faster"
+  homepage "https://github.com/ajeetdsouza/zoxide"
+  url "https://github.com/ajeetdsouza/zoxide/archive/v0.4.1.tar.gz"
+  sha256 "bf3bed2d384e2f137c1d4cf56a97fb7e925abd69823ce41f1552c76e183466c8"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args
```
It will put those args in place for you 😉

Reply:
Fixed!